drivers/docker/driver.go

```diff
@@ -1109,9 +1109,17 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 	}
 	hostConfig.Privileged = driverConfig.Privileged
 
+	// get docker client info (we need to know the runtime to adjust
+	// OS-specific capabilities)
+	client, err := d.getDockerClient()
+	if err != nil {
+		return c, err
+	}
+	ver, _ := client.Version()
```

Comment:
We should probably return this error rather than swallowing it, as the reason for the resulting behavior of a possibly-broken list of capabilities won't be observable to the user.

Reply:
Yeah my thinking was: if we can't get the version object for some reason, the Windows code will just play it safe and assume we're not using Mirantis, but you're right: makes it harder to debug.